steps/household/rent.tsx

```diff
@@ -97,7 +97,7 @@ const CurrentBalance: React.FunctionComponent = () => {
   const tenancyUrl = process.env.TENANCY_URL;
 
   const currentBalanceValue = currentBalance.result
-    ? `£${currentBalance.result}`
+    ? `£${Math.abs(parseFloat(currentBalance.result))}`
```

Comment:
We don't want to absolute it. If it's meant to be negative, it should render with a `-`. I think you want something like:


```suggestion
    ? `${Math.sign(parseFloat(currentBalance.result)) > 0 ? "- " : ""}£${currentBalance.result}`
```

Though I think we probably want to refactor this to self-document that what we're doing here is flipping the sign.

Reply:
Hello @erbridge , Gill did want this to be always positive like it is in outsystems as the word arrears tells them that it's negative. do you want me to check with her?

Reply:
I spoke to Gill, I misunderstood so I will make the change as suggested.